src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/repository/referral/ReferralResolutionRepository.java

```diff
@@ -5,5 +5,5 @@
 import uk.gov.justice.hmpps.datacompliance.repository.jpa.model.referral.ReferralResolution;
 
 @Repository
-public interface ReferralResolutionRepository extends CrudRepository<ReferralResolution, Long> {
+public interface ReferralResolutionRepository extends CrudRepository<ReferralResolution, Long>, ReferralResolutionRepositoryCustom {
```

Comment:
This follows the repository customisation approach here: https://docs.spring.io/spring-data/data-commons/docs/2.4.0-SNAPSHOT/reference/html/#repositories.custom-implementations allowing access to the EntityManager lock method.